my_list.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] [OPTIONAL] Because the previous CR didn't mention this it is not a requirement, however in the `each` method you created you are using the existing `each` method however the point of the assignment is to create the method from scratch using a loop. please consider trying to add this to your project as it is good practice.